### PR TITLE
Update Pods [--no-repo-update]

### PR DESCRIPTION
**Pull request by qzs21**
`pod update` needs to access the network. When the network is not good, need to wait for a long time.  And with `--no-repo-update`, it can save the network requests, in most cases we do not need to update the warehouse, we can quickly complete!

### CODE REVIEW

File: cocoadocs/CocoaPods.m [hunk 1]
```diff
@@ -102,6 +103,10 @@ - (void)addMenuItems
 		                                           keyEquivalent:@""];
 		self.installDocsItem.state = [self shouldInstallDocsForPods] ? NSOnState : NSOffState;
         
+        self.updatePodsNoRepoUpdateItem = [[NSMenuItem alloc] initWithTitle:@"Update Pods [--no-repo-update]"
```
Comment (kattrali) [inline cocoadocs/CocoaPods.m]: I think this should be a little more user-friendly, maybe "Update Pods (Offline only)"

**Comment by kattrali** [comment]
Looks good! Just the one change and I'll bring it in.

File: cocoadocs/CocoaPods.m (hunk 1 above)
Comment (qzs21) [inline cocoadocs/CocoaPods.m]: That great.

**Comment by kattrali** [comment]
[Merged!](https://github.com/kattrali/cocoapods-xcode-plugin/commit/c99e339dd61d142d7bb2121a758e84435d621b66) Thank you!

**Pull request closed without merging**